docs/deployment-guide.md

```diff
@@ -2,16 +2,149 @@
 
-## The control node
+## Getting started quickly with Vagrant
+
+The quickest way to get started and experiment with the playbook is by leveraging Vagrant to create a Virtualbox virtual machine to act as the control node **and** the target host.
+
+1. Ensure your local machine has a minimum of 10G of memory and 4 CPUs
+2. Clone via Git or Download this repository to your local machine
+3. Install [Vagrant](https://www.vagrantup.com/downloads)
+4. Install [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+5. Open Virtualbox application to make sure it startup correctly
+6. In a terminal, navigate to where you cloned or unpacked the repository
+7. Set environment variables to hold your Nexus credentials as shown below (replacing the values appropriately):
+
+    ```bash
+    export NEXUS_USERNAME="<your-username>"
+    export NEXUS_PASSWORD="<your-password>"
+    ```
+
+8. Run the main vagrant:
+
+    ```bash
+    vagrant up
+    ```
+
+    > NOTE: The playbook takes around 30 minutes to complete and mostly depends on your internet connection speed.
+
+Once ACS has initialized access the system using the following URLs using a browser:
+
+* Digital Workspace: `http://192.168.56.100/workspace`
+* Share: `http://192.168.56.100/share`
+* Repository: `http://192.168.56.100/alfresco`
+* API Explorer: `http://192.168.56.100/api-explorer`
+
+To access the machine vagrant created and ran the playbook on use `vagrant ssh`.
+
+## Getting started with localhost deployment
+
+If you have access to a pristine host running one of the supported Linux distributions, you can follow this quickstart for a localhost deployment.
+
+### Download playbook
+
+If you're not working directly working on the control node, transfer the ZIP
+file to the control node together with the SSH private key required to login to
+the target machines, and SSH into the machine:
+
+```bash
+scp  alfresco-ansible-deployment-<version>.zip user@controlnode:
+scp  -i ~/.ssh/ansible_rsa user@controlnode:.ssh
+ssh  user@controlnode
+unzip alfresco-ansible-deployment-<version>.zip
+cd alfresco-ansible-deployment
+```
+
+You can also use Git to fetch latest sources (or a specific release for example by adding `-b v2.1.0`) on the control node with:
+
+```bash
+git clone https://github.com/Alfresco/alfresco-ansible-deployment.git
+cd alfresco-ansible-deployment
+```
+
+> You may want to generate an SSH key pair locally and use it later for
+> deployment. Wether you generate one or you use one you copied over to the
+> control node, it is your responsibility to deploy it to the target machines so
+> Ansible can use it. Using SSH keys is recommended but not mandatory. If using
+> password instead make sure to add the `-k`switch to the ansible command so it
+> prompts you for a password.
+
+### Setup runtime environment
+
+Before starting using the playbook, make sure you are running at least python 3.8:
+
+```bash
+python3 --version
```

Comment:
changed all python to python3 because seems more reliable (I've tested on a bare ubuntu:22.04 container and `python` symlink is not present unless a specific package is installed)